### ISSUE TITLE
the raw/endraw do not seem to be working

### ISSUE THREAD

**Issue by podollb** [casid/jte#169]
If I have a template file `hi.sh.kte`
```
#!/bin/bash
@raw
echo "hi ${name}"
@endraw
```

I create my template engine and render:
```
TemplateEngine.createPrecompiled(ContentType.Plain)
templateEngine.render("hit.sh.kte")
```

I run into:
```
* What went wrong:
Execution failed for task ':precompileJte'.
> Failed to compile template, error at hi.sh.kte:4
                jteOutput.writeContent("\necho \"hi ${name}\"\n")
  jte-classes/gg/jte/generated/precompiled/JteentrypointshGenerated.kt:8:41
  Reason: Unresolved reference: name
```

I've tried various versions (even the latest) and no luck. What am I missing?

**Comment by casid** [comment]
Welcome @podollb and thanks for reporting!

This was fixed in this commit (https://github.com/casid/jte/commit/1d7d63119687674ffacb6ab592f633dfe76d1b92), the problem was, that `$` was not escaped in Kotlin strings. It should work with the latest snapshot version. You can try with this Jitpack build: https://jitpack.io/#casid/jte/d0a9b4a00c

I'm planning a new jte version release next week.

**Comment by casid** [comment]
I just released version [2.1.2](https://github.com/casid/jte/releases/tag/2.1.2) to maven central 👍
